UI/window-basic-main-outputs.cpp

```diff
@@ -2151,7 +2212,12 @@ bool AdvancedOutput::StartStreaming(obs_service_t *service)
 #endif
 	bool enableDynBitrate =
 		config_get_bool(main->Config(), "Output", "DynamicBitrate");
-
+	bool is_mpegts = false;
+	obs_service_t *service_obj = main->GetService();
+	const char *url = obs_service_get_connect_info(
+		service_obj, OBS_SERVICE_CONNECT_INFO_SERVER_URL);
+	if (url != NULL)
+		is_mpegts = isMpegts(url);
```

Comment:
You could just check the protocol that the service return and not the server URL.

Reply:
ah yeah good point. I haven't adjusted yet to the new api

Reply:
done